packages/evolution-backend/src/services/logging/queryLoggingMiddleware.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2023, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import { UserAttributes } from 'chaire-lib-backend/lib/services/users/user';
+import { _isBlank } from 'chaire-lib-common/lib/utils/LodashExtensions';
+import { NextFunction, Request, Response } from 'express';
+import interviewsAccessesDbQueries from '../../models/interviewsAccesses.db.queries';
+
+export type InterviewLoggingMiddlewares = {
+    openingInterview: (validationMode: boolean) => (req: Request, res: Response, next: NextFunction) => void;
+    updatingInterview: (validationMode: boolean) => (req: Request, res: Response, next: NextFunction) => void;
+};
+
+const openingInterview = (validationMode: boolean) => (req: Request, _res: Response, next: NextFunction) => {
```

Comment:
What is the validationMode? What does true/false means here? It's no obvious, would need comments